### PR TITLE
Register YourDJ.is-a.dev

### PR DESCRIPTION
**Pull request by TheDJGamerYT**
Register YourDJ.is-a.dev with CNAME record pointing to {   "record": {     "CNAME": " TheDJGamerYT.github.io"   } }.

### CODE REVIEW

File: .github/workflows/static.yml
Comment (DEV-DIBSTER) [inline .github/workflows/static.yml:1]: This file should not exist. Not sure why you're trying to mess with the workflows.

File: CNAME
Comment (DEV-DIBSTER) [inline CNAME:1]: This file also should not exist.

File: domains/yourdj.json
Comment (DEV-DIBSTER) [inline domains/yourdj.json:1]: Invalid file format, you have a record.cname.record inside this JSON.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: .github/workflows/static.yml
Comment (TheDJGamerYT) [reply .github/workflows/static.yml:1]: oh shit wrong registry lemme fix that

Comment (TheDJGamerYT) [reply .github/workflows/static.yml:1]: > This file should not exist. Not sure why you're trying to mess with the workflows.

i ment to put that iin a different registry
wasnt trying to fuck with it i was trying to make a github pages site with the help of a friend

Comment (TheDJGamerYT) [reply .github/workflows/static.yml:1]: and i used the discord bot automatic thingy

**Pull request closed without merging**